### DISCUSSION TITLE
Add team member to multiple sites at once

### DISCUSSION THREAD

**Discussion by RobertJoonas** [plausible/analytics#1350] [Anything Else]
Say I have 40 sites and I want to add a team member to all those sites. Currently, I would have to go through all my sites' settings and manually invite that user.

It would be nice if I could do that with one action by selecting a set of sites and inviting the user to all those.

(feedback via email)

**Comment by ukutaht** [comment]
I'm planning to build something like a 'team account' where a team could own sites so team members don't have to be added one-by-one. Not sure about this case but I think for most teams it's a better solution.

**Reply by Staffan-FNX** [reply]
Would be very valuable. If combined with grouping of sites even better!

**Reply by jonginn** [reply]
Am looking at using Plausible for 30+ sites and being able to assign a Team to specific sites - and add people to that Team later - would be a key feature.

**Reply by thechelsuk** [reply]
+1 for this, also add it's not just adding users a key use case in my organization is where a member of the team leaves. Four our >60 sites thats a real pain to go one by one to remove the user.

Many thanks

**Reply by hamishcampbell** [reply]
Agree with above, team management would be a big win. In the interim, even a simple user management API would help.

**Reply by 3stacks** [reply]
Big +1 on this. My company has heaps of very short-lived sites so I keep forgetting to add relevant people to new sites

**Reply by hamishcampbell** [reply]
@ukutaht Hey wondering if you made any progress to this? It's the main blocker to us adopting Plausible more broadly. It's also a problem for managing security policies - so despite Plausible getting a big tick on handling PII / GDPR / CCPA, the difficulty of managing user access makes it less attractive when you need to scale up.

**Reply by metmarkosaric** [reply]
hi @hamishcampbell! no progress on this for now. we're currently working on features such as custom properties for pageviews, monetary values for goals and funnels. we'd like to make our team accounts easier to manage in the future but these product gaps are more important to bridge right now. currently you can add or remove members one at a time and one site at a time. as soon as we have some more convenient way to add multiple people to multiple site at once, we will update this thread. thanks!

**Reply by bryevdv** [reply]
+1 for better teams/account management. I was recently asked to write up our experience migrating to Plausible from GA for another team, and while my summary was extremely positive, I reported that lack of basic teams/account features was the biggest negative (by far).

**Comment by bogplau** [comment]
Right now, you need to invite users to each specific property, which makes it harder to have an overview of who has access to what property. If you would introduce a users screen that would list all users with properties they have access to and allow to directly add a single user to multiple properties or modify their access that would be great. It would also simplify the process of removing access for a single user from multiple properties.

(received via email)

**Comment by hamishcampbell** [comment]
Hello again, wondering if Team management was on the radar yet. This is really the only thing stopping us (and I suspect a lot other people) using Plausible a heck of a lot more.

**Comment by KaelWD** [comment]
On self-hosted there's also nothing stopping a user from creating new sites and not sharing them.

**Comment by JinsoRaj** [comment]
It is really painful to add all the team members to the sites individually, kindly add this feature.

**Comment by sdarwin** [comment]
- Taking an example from AWS, GCP, Active Directory, etc..   how about adding the concept of "groups".   Place multiple users into a group and then allow an entire group to have permissions on a website.    

- Another distinct setting could be "initial default users".   Upon creation of a new website in Plausible, the website inherits the "initial default users and groups".

**Comment by Hricha-Shandily** [comment]
This is now possible to do in Plausible, all info here: https://plausible.io/docs/users-roles
